packages/reassure-compare/src/types.ts

```diff
@@ -1,47 +1,11 @@
 /** Parsed performance results file. */
+import type { PerformanceEntry, PerformanceMetadata } from './utils/validate';
+
 export interface PerformanceResults {
   metadata?: PerformanceMetadata;
   entries: Record<string, PerformanceEntry>;
 }
 
-/** Header of performance results file. */
-export interface PerformanceHeader {
-  metadata: PerformanceMetadata;
-}
-
-/** Metadata information for performance results. */
-export interface PerformanceMetadata {
-  branch?: string;
-  commitHash?: string;
-}
-
-/** Entry in the performance results file. */
-export interface PerformanceEntry {
-  /** Name of the test scenario */
```

Comment:
Can we save the code comments and moved them over respective Zod schema definitions for all the types and fields involved?